librustzcash/Cargo.toml

```diff
@@ -28,6 +28,7 @@ libc = "0.2"
 pairing = { version = "0.15.0", path = "../pairing" }
 lazy_static = "1"
 rand_core = "0.5.1"
+zcash_mmr = { git = "https://github.com/nikvolf/zcash-mmr" }
```

Comment:
I think we should put the git revision here, so that if `master` updates upstream it won't cause problems.

```
zcash_mmr = { git = "https://github.com/nikvolf/zcash-mmr", rev = "e2c131fdc308265adcab774e54d4d5804c23b368" }
```

Reply:
We'll be vendoring the crate really soon so this isn't really a problem.